### PR TITLE
readme 12

### PR DESCRIPTION
**Pull request by nikhilgarg1047**
readme 12

### CODE REVIEW

**Comment by nikhilgarg1047** [comment]
c101

**Comment by nikhilgarg1047** [comment]
c102

File: README.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ ewdwe
 ercwerc
 qwedqwe
 dcqwed
+wedwdecq
```
Comment (nikhil7857851) [inline README.md:9]: Why this change?

**Review by nikhil7857851 — APPROVED** [review]

**Review by nikhil7857851 — COMMENTED** [review]
Ok

**Comment by nikhilgarg1047** [comment]
c121
